test/fit_tests/common/fit_common.py

```diff
@@ -703,18 +707,27 @@ def apply_obm_settings_new():
                             nodestatus[node]['status'] = "succeeded"
                         if state in ["failed", "cancelled", "timeout"]:
                             nodestatus[node]['status'] = "pending"
-        if VERBOSITY > 4:
+                            # if the workflow left an invalid OBM, delete it
+                            result = rackhdapi("/api/2.0/nodes/" + node)
+                            if result['status'] == 200:
+                                if result['json']['obms']:
+                                    for ref in result['json']['obms']:
+                                        obmref = ref.get('ref')
```

Comment:
@gpaulos To me it looks like you are pulling the `obms` property out of a node.  I don't understand the `ref.get('ref') part.  What are you looking at here.  None of the obms should have a `ref` property.